### ISSUE TITLE
Using driftscan to do power spectrum estimation 

### ISSUE THREAD

**Issue by Zahra-Kader** [UKZN-Astronomy/hirax_corrcal#5]
@dcrichton Hi Devin
Would you be able to share the config file to run driftscan for PS estimation?

**Comment by piyanatk** [comment]
From #10, power spectrum estimation is part of `draco`, correct?

I think we want this to part of the `draco-corrcal-pipeline` notebook.

**Comment by Zahra-Kader** [comment]
Yes, ps estimation is done using draco, as shown in the config_draco_ps.txt file attached in #10
